src/partition_table.toit

```diff
@@ -244,14 +248,19 @@ class PartitionTable:
 
       offset-string := parts[3]
       offset/int := ?
-      if offset-string.starts-with "0x": offset = int.parse offset-string[2..] --radix=16
+      if offset-string == "":
+        if not next-computed-offset: throw "Missing initial offset"
+        offset = next-computed-offset
+      else if offset-string.starts-with "0x": offset = int.parse offset-string[2..] --radix=16
       else: offset = int.parse offset-string
```

Comment:
Now that the first then-block is indented, I think I prefer to indent the next ones too.

```suggestion
      if offset-string == "":
        if not next-computed-offset: throw "Missing initial offset"
        offset = next-computed-offset
      else if offset-string.starts-with "0x": 
        offset = int.parse offset-string[2..] --radix=16
      else: 
        offset = int.parse offset-string
```